content/faq/naming.md

```diff
@@ -26,29 +26,27 @@ So we thought, “What if there is a better way?” Consulting with economists,
 | **Channel** | `lbry://<@channel_name>` | A claim containing a public key, the private counterpart is used for signing other name claims. Channel claims can be specified in the uri with the `#` modifier, and the unmodified uri has vanity resolution. |
 | **Signed** | `lbry://<@channel_name>/<example>` | The claim to `<example>` in the channel of (signed by) `<@channel_name>`. The channel claim defaults to the vanity resolution, but can be specified with the `#` modifier given before the `/`.
 
-More details on how to publish to premanent, vanity, channel, and signed.
+#### What is a claim?</br>
 
-What is a claim?</br>
+Claims are vanity names will be controlled by the people who get the most value out of them. For example, Radiohead would get a lot more value out of lbry://amoonshapedpool than a squatter, pirate, or troll.
```

Comment:
"Claims are vanity names will be" ? Probably not good to start talking about vanity claims before explaining what a claim is.